content/docs/started/getting-started-k8s.md

```diff
@@ -144,11 +144,342 @@ kubectl get cm dex -n kubeflow -o jsonpath='{.data.config\.yaml}' > dex-config.y
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: phpldapadmin
+          template:
+            metadata:
+              labels:
+                app: phpldapadmin
+            spec:
+              containers:
+                - name: phpldapadmin
+                  image: osixia/phpldapadmin:0.8.0
+                  ports:
+                    - name: http-server
+                      containerPort: 80
+                  env:
+                    - name: PHPLDAPADMIN_HTTPS
+                      value: "false"
+                    - name: PHPLDAPADMIN_LDAP_HOSTS
+                      value : "#PYTHON2BASH:[{'ldap-service.kubeflow.svc.cluster.local': [{'server': [{'tls': False}]},{'login': [        {'bind_id': 'cn=admin,dc=arrikto,dc=com'}]}]}]"
+        {{< /highlight >}}
+        
+        </details>
+        
+        
+    2. Seed the LDAP database with new entries.
+        
+        ```bash
+        kubectl exec -it -n kubeflow ldap-0 -- bash
+        ldapadd -x -D "cn=admin,dc=arrikto,dc=com" -W
+        # Enter password "admin".
+        # Press Ctrl+D to complete after pasting the snippet below.
+        ```
+        
+        <details>
+        
+        <summary>LDAP Seed Users and Groups</summary>
+        ```ldif
+        # This already exists.
+        # If it doesn't, uncomment this.
+        #dn: dc=arrikto,dc=com
+        #objectClass: dcObject
+        #objectClass: organization
+        #o: Arrikto
+        #dc: arrikto
+        
+        dn: ou=People,dc=arrikto,dc=com
+        objectClass: organizationalUnit
+        ou: People
+        
+        dn: cn=Yannis Zarkadas,ou=People,dc=arrikto,dc=com
+        objectClass: person
+        objectClass: inetOrgPerson
+        givenName: Yannis
+        sn: Zarkadas
+        cn: Yannis Zarkadas
+        uid: yanniszark
+        mail: yanniszark@arrikto.com
+        userpassword: 12341234
+        
+        dn: cn=Vangelis Koukis,ou=People,dc=arrikto,dc=com
+        objectClass: person
+        objectClass: inetOrgPerson
+        givenName: Vangelis
+        sn: Koukis
+        cn: Vangelis Koukis
+        uid: vkoukis
+        mail: vkoukis@arrikto.com
+        userpassword: 43214321
+        
+        # Group definitions.
+        
+        dn: ou=Groups,dc=arrikto,dc=com
+        objectClass: organizationalUnit
+        ou: Groups
+        
+        dn: cn=admins,ou=Groups,dc=arrikto,dc=com
+        objectClass: groupOfNames
+        cn: admins
+        member: cn=Yannis Zarkadas,ou=People,dc=arrikto,dc=com
+        
+        dn: cn=developers,ou=Groups,dc=arrikto,dc=com
+        objectClass: groupOfNames
+        cn: developers
+        member: cn=Yannis Zarkadas,ou=People,dc=arrikto,dc=com
+        member: cn=Vangelis Koukis,ou=People,dc=arrikto,dc=com
+        ```
+        
+        </details>
+
+2. To use your LDAP/AD server with Dex, you have to edit the Dex config.To edit the ConfigMap containing the Dex config, run:
```

Comment:
Change "run" to "follow these steps" - because the reader expects a command to follow the word "run", but instead comes across some steps with text.